src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -196,13 +199,44 @@ function (Document $document) use ($dbForConsole, $getProjectDB) {
         );
     }
 
+    /**
+     * @param Document $project
+     * @param callable $getProjectDB
+     * @param Document $provider
+     * @throws Exception
+     */
+    protected function deleteProvider(Document $project, callable $getProjectDB, Document $provider)
```

Comment:
Since we are going to allow different providers for a single topic, we can't delete topics here. Let's just delete the messages

Reply:
removed this altogether, messages will also not be tied to a single provider anymore